presto-kafka/src/main/java/io/prestosql/plugin/kafka/schema/file/FileTopicSchemaProviderModule.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.plugin.kafka.schema.file;
+
+import com.google.inject.Binder;
+import com.google.inject.Scopes;
+import io.airlift.configuration.AbstractConfigurationAwareModule;
+import io.prestosql.plugin.kafka.schema.TopicSchemaProvider;
+
+import static com.google.inject.multibindings.MapBinder.newMapBinder;
+import static io.airlift.configuration.ConfigBinder.configBinder;
+
+public class FileTopicSchemaProviderModule
+        extends AbstractConfigurationAwareModule
+{
+    @Override
+    protected void setup(Binder binder)
+    {
+        configBinder(binder).bindConfig(FileTopicSchemaProviderConfig.class);
+        newMapBinder(binder, String.class, TopicSchemaProvider.class).addBinding(FileTopicSchemaProvider.NAME).toProvider(FileTopicSchemaProvider.class).in(Scopes.SINGLETON);
```

Comment:
this is not needed. Just bind `TopicSchemaProvider` to `FileTopicSchemaProvider`